etc/init.d/zfs-mount.in

```diff
@@ -13,10 +13,11 @@
 #
 ### BEGIN INIT INFO
 # Provides:          zfs-mount
-# Required-Start:    $local_fs zfs-import
+# Required-Start:    mountall zfs-import
```

Comment:
Does this mean `mountall` command is ran before zfs-mount? If so, that seems incorrect as we should mount all ZFS datasets before processing fstab entries, as done in zfs-mount service.

Reply:
Yes. Currently it is one of the **last** things to happen on boot. I wasn’t sure if we should mount before or after the local fs. I guess it really depends on how your fs layout is setup. I can move it up sooner if desired.

Reply:
I'm thinking we don't need to specify `mountall`, starting zfs-import service early, as you have it now, should be sufficient.